spec/models/bundle_context_spec.rb

```diff
@@ -17,64 +17,44 @@
       expect(BundleContext.new).not_to be_valid
       expect(bc).to be_valid
     end
-
-    context "defines enum with expected values" do
-      it "content_structure enum" do
-        is_expected.to define_enum_for(:content_structure).with(
-          "simple_image" => 0,
-          "simple_book" => 1,
-          "book_as_image" => 2,
-          "file" => 3,
-          "smpl" => 4
-        )
-      end
-
-      it "content_metadata_creation enum" do
-        is_expected.to define_enum_for(:content_metadata_creation).with(
-          "default" => 0,
-          "filename" => 1,
-          "smpl_cm_style" => 2
-        )
-      end
+    it 'is not valid without a User' do
+      expect { bc.user = nil }.to change(bc, :valid?).to(false)
     end
-
-    describe "#content_structure=" do
-      it "validation rejects a value if it does not match the enum" do
-        expect { described_class.new(content_structure: 654) }
-          .to raise_error(ArgumentError, "'654' is not a valid content_structure")
-        expect { described_class.new(content_structure: 'book_as_pdf') }
-          .to raise_error(ArgumentError, "'book_as_pdf' is not a valid content_structure")
-      end
-
-      it "will accept a symbol, but will always return a string" do
-        expect(described_class.new(content_structure: :smpl).content_structure).to eq 'smpl'
-      end
+    it 'is not valid unless bundle_dir exists on filesystem' do
+      expect { bc.bundle_dir = 'does/not/exist' }.to change(bc, :valid?).to(false)
     end
+  end
 
-    describe "#content_metadata_creation=" do
-      it "validation rejects a value if it does not match the enum" do
-        expect { described_class.new(content_metadata_creation: 654) }
-          .to raise_error(ArgumentError, "'654' is not a valid content_metadata_creation")
-        expect { described_class.new(content_metadata_creation: 'dpg') }
-          .to raise_error(ArgumentError, "'dpg' is not a valid content_metadata_creation")
-      end
-
-      it "will accept a symbol, but will always return a string" do
```

Comment:
These tests were testing how rails enum works, not how the app uses them.  When the enum is set, the attribute getter always returns one of the enumerated values, exactly as they were enumerated.